com.unity.render-pipelines.high-definition/Editor/Material/ShaderGraph/SurfaceSubTarget.cs

```diff
@@ -248,18 +233,14 @@ public override void GetFields(ref TargetFieldContext context)
             context.AddField(HDFields.DoAlphaTestPostpass,  systemData.alphaTest && systemData.alphaTestDepthPostpass && isTransparentDepthPostpass &&
                                                             context.pass.validPixelBlocks.Contains(HDBlockFields.SurfaceDescription.AlphaClipThresholdDepthPostpass));
 
-
-            context.AddField(HDFields.TransparentDepthPrePass,      systemData.surfaceType != SurfaceType.Opaque && systemData.alphaTestDepthPrepass);
-            context.AddField(HDFields.TransparentDepthPostPass,     systemData.surfaceType != SurfaceType.Opaque && systemData.alphaTestDepthPostpass);
-
             // Features & Misc
             context.AddField(Fields.LodCrossFade,                   systemData.supportLodCrossFade);
-            context.AddField(Fields.VelocityPrecomputed,            builtinData.addPrecomputedVelocity);
-            context.AddField(HDFields.TransparentWritesMotionVec,   systemData.surfaceType != SurfaceType.Opaque && builtinData.transparentWritesMotionVec);
             context.AddField(Fields.AlphaToMask,                    systemData.alphaTest && context.pass.validPixelBlocks.Contains(BlockFields.SurfaceDescription.AlphaClipThreshold) && builtinData.alphaToMask);
-            context.AddField(HDFields.DepthOffset,                  builtinData.depthOffset && context.pass.validPixelBlocks.Contains(HDBlockFields.SurfaceDescription.DepthOffset));
```

Comment:
We must let all the field here, they will be the default value in the future. Also it is required for people to test in the preview

Reply:
Fields are only used to control what is generated in the template right ? for the default value we have a property that is added here: https://github.com/Unity-Technologies/Graphics/pull/952/files#diff-6da0f4a431c6099d9b46856b1bf6d93cR331
And it will also affect the preview because we call the reset keyword function on the preview material which will disable/enable the _DEPTHOFFSET_ON keyword based on the depth offset property value